sql/rename.go

```diff
@@ -203,6 +203,88 @@ func (p *planner) RenameIndex(n *parser.RenameIndex) (planNode, error) {
 
 	idx.Name = newIdxName
 	descKey := MakeDescMetadataKey(tableDesc.GetID())
+	if err := tableDesc.Validate(); err != nil {
```

Comment:
I don't think there's anything you can pass that will reach Validate() and not be caught by your code first.